vector-stores/spring-ai-hanadb-store/README.md

```diff
@@ -0,0 +1,284 @@
+# SpringAI using SAP HANA Cloud vector engine
```

Comment:
Please convert the README into an adoc under the [vectordbs](https://github.com/spring-projects/spring-ai/tree/main/spring-ai-docs/src/main/antora/modules/ROOT/pages/api/vectordbs) and add a link to the [nav.adoc](https://github.com/spring-projects/spring-ai/blob/main/spring-ai-docs/src/main/antora/modules/ROOT/nav.adoc).  
The README can contain a single line link to the adoc documentation

Reply:
+ACK